golos.vesting/golos.vesting.cpp

```diff
@@ -552,8 +550,6 @@ void vesting::providebw_for_trx(transaction& trx, const permission_level& provid
 }
 
 void vesting::timeout(symbol symbol) {
-    require_auth(_self);
```

Comment:
looks like this allows anybody to spam with `timeout` for free (using free `providebw`) till provider have resources

Reply:
the same: you can create unlimited comments using free providebw.
its a problem of provider.

Reply:
yup, it's not `requestbw`, so ok